templates/4a_compensate_image_data_round2.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
<p>We don't need to keep track of roseeta_2 coefficients. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>